src/user-icon/user-icon.scss

```diff
@@ -91,23 +91,24 @@
       box-shadow: inset 0 0 0 1px rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
     }
 
-    > .iui-status-symbol {
-      display: inline-flex;
-      @include themed {
-        fill: t(iui-icons-color);
-      }
+    &::after {
+      content: '';
+      mask: url("data:image/svg+xml,<svg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'><path d='m12.5 2-4.5 4.5-4.5-4.5-1.5 1.5 4.5 4.5-4.5 4.5 1.5 1.5 4.5-4.5 4.5 4.5 1.5-1.5-4.5-4.5 4.5-4.5z' /></svg>");
```

Comment:
Previously, we could have passed what ever icon there, now there is always gonna be `offline` one by default...
Should we add class for offline icon too?